cmd/fleet/cron.go

```diff
@@ -226,6 +242,20 @@ func cronVulnerabilities(
 						level.Error(logger).Log("err", "queueing vulnerabilities to jira", "details", err)
 						sentry.CaptureException(err)
 					}
+
+				case "zendesk":
+					// queue job to create zendesk ticket
+					if err := worker.QueueZendeskJobs(
+						ctx,
+						ds,
+						kitlog.With(logger, "zendesk", "vulnerabilities"),
+						recentVulns,
+					); err != nil {
+						level.Error(logger).Log("err", "queueing vulnerabilities to jira", "details", err)
+						sentry.CaptureException(err)
+					}
+
+				default:
```

Comment:
While you are at it, to prevent future bugs add a return error in the `default` case.